src/main/resources/doc/de/html/guide/about/indexa.html

```diff
@@ -12,7 +12,7 @@
     <link rel="stylesheet" type="text/css" href="../../style.css">
   </head>
   <body>
-    <div class=maindiv>
+    <div class="maindiv">
```

Comment:
Could you please do a separate PR where only this change is done?
When we merge it, this PR will become much cleaner and easier to review.

Reply:
thank you for your advice